src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -788,4 +786,9 @@ private AnomalyDetectorSettings() {}
     // such as "there are at least 10000 entities", the default is set to 10,000. That is, requests will count the
     // total entities up to 10,000.
     public static final int MAX_TOTAL_ENTITIES_TO_TRACK = 10_000;
+
```

Comment:
"MAX_TOTAL_ENTITIES_TO_TRACK = 10_000;" seems like an upper bound. Should the variable be renamed re "MAX_BUCKETS"?? Or are we limiting to 10000 for now (which can be a deliberate decision as well)?

Reply:
It is Yaliang's code :) I'll leave the decision to @ylwu-amzn 

We are limiting to 10000 in historical detectors because historical detectors are more heavier than real time.  I guess it can be changed to a dynamic setting and let users configure.  Will leave it to Yaliang.

Reply:
Ok. Makes sense. Let's not make hasty changes.

Reply:
@kaituo , seems `MAX_TOTAL_ENTITIES_TO_TRACK` is for entity profile of realtime HCAD, this is written by you, in [PR182](https://github.com/opensearch-project/anomaly-detection/pull/182/files#diff-dfb14a8e1cf93e3abc10cb12cdb3a9b0271728f9b8a8e1dd3c4868697ef7816dR795)

Reply:
oh, you are right.  I slept too little these days :) @ylwu-amzn